go/ql/lib/semmle/go/security/CommandInjectionCustomizations.qll

```diff
@@ -29,8 +29,13 @@ module CommandInjection {
    */
   abstract class Sanitizer extends DataFlow::Node { }
 
+  /**
+   * DEPRECATED: Use `RemoteFlowSource` or `Source` instead.
+   */
+  deprecated class UntrustedFlowAsSource = RemoteFlowAsSource;
+
   /** A source of untrusted data, considered as a taint source for command injection. */
-  class UntrustedFlowAsSource extends Source instanceof UntrustedFlowSource { }
+  private class RemoteFlowAsSource extends Source instanceof RemoteFlowSource { }
```

Comment:
Why `private`?

Reply:
It just doesn't make much sense to make this public. It isn't really of any use for external code to reference it, and will probably just lead to confusion if it is done by mistake.